lib/js-api/src/api/gs-core-rest-api/model/hold.ts

```diff
@@ -16,9 +16,9 @@
  */
 
 export interface Hold {
-    id?: string;
-    name?: string;
-    reason?: string;
+    id: string;
+    name: string;
+    reason: string;
```

Comment:
Should reason be required? I think it's not returned by API

Reply:
when we are creating new hold it is required, so I think it should be also in model

Reply:
As far as I can see `HoldBody` is used for creating holds so I don't think it should be required especially that api does not return it

Reply:
discussed, split interfaces